.github/workflows/ci.yml

```
@@ -27,6 +27,142 @@ jobs:
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+      - name: Download katana binary
+        uses: actions/download-artifact@v4
+        with:
+          name: dojo-bins
+          path: /tmp/bins
+
+      - run: chmod +x /tmp/bins/katana
+
+      - name: Start katana nodes
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:14:3: Double quote to prevent globbing and word splitting

(shellcheck)